tested/evaluators/exception.py

```diff
@@ -67,9 +69,10 @@ def evaluate(_config: EvaluatorConfig,
 
     return EvaluationResult(
         result=StatusMessage(
-            enum=Status.CORRECT if expected.message == actual.message else 
+            enum=Status.CORRECT if expected.message == actual.message else
             Status.WRONG
         ),
         readable_expected=expected.readable(),
-        readable_actual=actual.readable()
+        readable_actual=_config.bundle.lang_config.exception_output(actual)
```

Comment:
The `exception_output` is already done in `try_as_readable_exception`, and this variable is also the result of `try_as_readable_exception`, so this applies it twice

Reply:
Ah you're right, my bad, you can ignore my comment in that case